pydy/viz/scene.py

```diff
@@ -350,16 +497,15 @@ def generate_visualization_json_system(self, system, **kwargs):
         ``generate_visualization_json`` method.
 
         """
-        if not isinstance(system, System):
-            msg = "{} should be a valid pydy.System object".format(system)
-            raise TypeError(msg)
-        else:
-            self._system = system
 
-        self.generate_visualization_json(system.states,
-                                         system.constants.keys(),
-                                         system.integrate(),
-                                         system.constants.values(), **kwargs)
+        self.system = system
+        prefix = None
+        for k, v in kwargs.items():
+            if k == 'fps':
+                self.frames_per_second = v
+            if k == 'outfile_prefix':
+                prefix = v
+        self._generate_json(prefix=prefix)
 
     def create_static_html(self, overwrite=False, silent=False):
```

Comment:
`create_static_html` should create a directory called `static` in the cwd and fill it with both the files from the `static` directory in the pydy source tree and the two json files for the current simulation (i.e. whatever the simulation is at the time `create_static_html` is called. Then the user has a single directory that they could copy to their website server and display the visualization online for others to view. This is the method that we can use to embed everything in the pydy website eventually.